README.md

```diff
@@ -1,22 +1,58 @@
 ViperMonkey
 ===========
 
+This is a fork of Philippe Lagadec's (http://www.decalage.info) ViperMonkey.
```

Comment:
Please remove this mention so that I can merge the PR.

Reply:
Fixed in commit c963b624223cefed44fbb08bf7e70bbca0a6bf54.